modules/installation-aws-permissions.adoc

```diff
@@ -212,19 +207,25 @@ If you have not created a load balancer in your AWS account, the IAM user also r
 * `s3:PutObject`
 * `s3:PutObjectAcl`
 * `s3:PutObjectTagging`
+
+[NOTE]
+=====
+If you set the `preserveBootstrapIgnition` AWS parameter in your `install-config.yaml` file, the `s3:DeleteObject` permission is optional.
```

Comment:
@dfitzmau  This description is correct, but currently, installer still checks if this permission exist [1], so we can add this note only if it's removed from the check list, or add a condition like [2], @patrickdillon WDYT?

[1] https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go#L177
[2] https://github.com/openshift/installer/pull/8296

Reply:
@dfitzmau @patrickdillon  How about we drop this note, and still consider `s3:DeleteObject` as required permission, as the `preserveBootstrapIgnition` is not enabled by default.

Reply:
@dfitzmau  I think we can hold this part until https://issues.redhat.com/browse/OCPBUGS-33509 gets resolved.

Reply:
Removed:

> If you set the `preserveBootstrapIgnition` AWS parameter in your `install-config.yaml` file, the `s3:DeleteObject` permission is optional.

This work will continue on the following Jiras:

- https://issues.redhat.com/browse/OCPBUGS-33661
- https://issues.redhat.com/browse/OCPBUGS-33662